ambari-infra/pom.xml

```diff
@@ -99,6 +99,28 @@
   <build>
     <pluginManagement>
       <plugins>
+       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>rpm-maven-plugin</artifactId>
+        <version>2.0.1</version>
```

Comment:
can you use 2.1.4 version here (and for logsearch)? as that one is used in the assembly module